common/v2/features/Dashboard/components/WalletBreakdown/AccountDropdown.tsx

```diff
@@ -113,7 +113,13 @@ const AccountDropdown = ({ accounts = [], selected = [], onSubmit }: AccountDrop
 
   // Only update our draft if the prop changed.
   // https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects
-  useEffect(() => setDraftSelected(selected), [selected]);
+  useEffect(() => {
+    // Filters out dashboard account uuids that have been deleted.
+    const filteredSelected = selected.filter(selectedAccountUUID => {
```

Comment:
This is hidding a more important bug.
- AccountDropdown is passed `accounts` & `selected`.
- it displays the length of both.
- `selected` corresponds to `Store/SettingsProvider` and when we delete an account we don't update our `settings`.

To solve the bug we could either enforce the coupling between settings and accounts or change the data structure. 
In any case I would prefer to keep the bug visible then to hide it behind a fix.